server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -311,15 +311,9 @@ public TSFetchMetadataResp fetchMetadata(TSFetchMetadataReq req) {
           resp.setMetadataInJson(metadataInJson);
           status = new TSStatus(getStatus(TSStatusCode.SUCCESS_STATUS));
           break;
-        case "DELTA_OBEJECT":
-          Metadata metadata = getMetadata();
-          String column = req.getColumnPath();
-          Map<String, List<String>> deviceMap = metadata.getDeviceMap();
-          if (deviceMap == null || !deviceMap.containsKey(column)) {
-            resp.setColumnsList(new ArrayList<>());
-          } else {
-            resp.setColumnsList(deviceMap.get(column));
-          }
+        case "SHOW_DEVICES":
+          Set<String> devices = getAllDevices();
+          resp.setShowDevices(devices);
```

Comment:
```suggestion
          resp.setDevices(devices);
```

Reply:
Here I also keep consistent with `SHOW TIMESERIES` and `SHOW STORAGE GROUP` interface, in which they uses `setShowTimeseriesList` and `setShowStorageGroups`... Should I change all of these interfaces? (and I also think your suggestion is better...)